src/plays/github-user-finder/utilities/formateDate.js

```diff
@@ -0,0 +1,10 @@
+export default function formatDate(data) {
```

Comment:
Why don't you move it under commonUtils? The file exists already.

Reply:
What should be the structure should i just copy paste my file there and then import it or there are some guidelines i hae to follow like defining function inside some common file and export it from there as named export.

Reply:
Just move the function there and import the function from that common file. @ANKIT9761

Reply:
If you have moved it, then delete the file.

Reply:
@atapas done with the changes